sample/src/main/java/co/nimblehq/sample/ui/surveylist/SurveyListViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package co.nimblehq.sample.ui.surveylist
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import co.nimblehq.sample.ui.ModelResult
+import co.nimblehq.survey.sdk.entity.SurveyEntity
+import kotlinx.coroutines.launch
+
+class SurveyListViewModel : ViewModel() {
+
+    private val _surveyListResult = MutableLiveData<ModelResult<SurveyListModel>>()
+    val surveyListResult: LiveData<ModelResult<SurveyListModel>> = _surveyListResult
+
+    fun getSurveyList() {
+        // can be launched in a separate asynchronous job
+        viewModelScope.launch {
+            try {
+                //TODO: hardcode for testing, need to be integrate with SDK later
+                _surveyListResult.value = ModelResult(success = SurveyListModel(listSurvey = mutableListOf(
```

Comment:
it's a minor thing, but in many projects, we call this `ModelResult` is `UiModel` - denote for the data we want to send back to View for displaying; perhaps you would like to consider follow our convention from now on? 🙋‍♂️

Reply:
for example: `SurveyUiModel`

Reply:
@sleepylee Good to know. I will follow our convention.